### PR TITLE
Feature/social media handlers

### PR DESCRIPTION
**Pull request by shihabus**
- Adding Lato
- Adding social media handler links

### CODE REVIEW

File: css/events.css [hunk 1]
```diff
@@ -1,5 +1,9 @@
+body {
+	font-family: 'Lato', sans-serif;
+}
+
```
Comment (ankushdharkar) [inline css/events.css:4]: `main.css` already has font. Can you update there please?

File: css/events.css [hunk 2]
```diff
@@ -9,81 +13,95 @@
...
 
+.social-media-logo {
+  width: 30px;
+  margin: 0px 4px;
+}
+
+.social-media-logo:hover {
+  transform: scale(1.1);
+}
+
```
Comment (ankushdharkar) [inline css/events.css:24]: Super! Thanks

File: events.html [hunk 3]
```diff
@@ -11,11 +11,14 @@
 	<link rel="stylesheet" href="./css/events.css" />
 	<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;0,500;0,700;1,400&display=swap"
 		rel="stylesheet" />
+	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;400;700&display=swap" rel="stylesheet">
```
Comment (ankushdharkar) [inline events.html:14]: Let's not keep both the fonts. Either Roboto Or Lato

File: events.html [hunk 4]
```diff
@@ -11,11 +11,14 @@
...
 </head>
 
 <body>
 	<header>
-		<img class="logo-img" src="img/Real-Dev-Squad@1x.png" alt="Real Dev Squad logo" />
+		<a href="https://realdevsquad.com/" target="_blank" rel="noopener noreferrer">
+			<img class="logo-img" src="img/Real-Dev-Squad@1x.png" alt="Real Dev Squad logo" />
+		</a>
```
Comment (ankushdharkar) [inline events.html:21]: Let's not open this in a new page?

File: events.html [hunk 5]
```diff
@@ -11,11 +11,14 @@
 	<link rel="stylesheet" href="./css/events.css" />
 	<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;0,500;0,700;1,400&display=swap"
 		rel="stylesheet" />
+	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;400;700&display=swap" rel="stylesheet">
 </head>
 
 <body>
 	<header>
-		<img class="logo-img" src="img/Real-Dev-Squad@1x.png" alt="Real Dev Squad logo" />
+		<a href="https://realdevsquad.com/" target="_blank" rel="noopener noreferrer">
```
Comment (ankushdharkar) [inline events.html:19]: `<a href="/" ...>` should be fine

File: css/main.css [hunk 6]
```diff
@@ -1,5 +1,5 @@
 body {
-  font-family: 'Roboto', sans-serif;
+  font-family: "Lato", sans-serif;
   /*
```
Comment (ankushdharkar) [inline css/main.css:3]: Let's not change the font in this PR

File: events.html (hunk 3 above)
Comment (ankushdharkar) [reply events.html:14]: Let's keep Roboto for now. Will change to Lato in another PR

File: css/main.css (hunk 6 above)
Comment (shihabus) [reply css/main.css:3]: ok

**Pull request merged**